subprojects/file-watching/src/main/java/org/gradle/internal/watch/registry/impl/WatchableHierarchies.java

```diff
@@ -183,6 +188,30 @@ private SnapshotHierarchy invalidateUnsupportedFileSystems(SnapshotHierarchy roo
         }
     }
 
+    @CheckReturnValue
+    private SnapshotHierarchy updateUnwatchableFilesOnBuildStart(SnapshotHierarchy root, Invalidator invalidator, WatchMode watchMode) {
+        SnapshotHierarchy newRoot = root;
+        // Make sure we don't have any unwatchable files in the VFS, since the some previously unwatchable files may become watchable.
```

Comment:
```suggestion
        // Make sure we don't have any unwatchable files in the VFS, since some previously unwatchable files may have become watchable
```

Reply:
I changed the comment.